test/e2e/jest-puppeteer.config.js

```diff
@@ -18,14 +18,13 @@ module.exports = {
     command: `npm run start -- --config-server-port 10002`,
     port: 10002,
     protocol: 'http', // if default or tcp, the test starts right await whereas the dev server is not available on http
-    launchTimeout: 30000, // high value mainly for GitHub Workflows running on macOS (slow machines)
+    launchTimeout: 120000, // high value mainly for GitHub Workflows running on macOS (slow machines) and to build the bundle before start
```

Comment:
❓ can we put a lower value like 60000? waiting for 2 minutes for the server to start (including dev bundle build) seems to much, 1 minute should be enough

Reply:
I'll try 😉

Reply:
Done